samcli/commands/init/interactive_init_flow.py

```diff
@@ -349,6 +350,20 @@ def get_sorted_runtimes(runtime_option_list):
     return sorted_runtime
 
 
+def get_supported_runtime(runtime_list):
+    supported_runtime_list = []
+    error_message = ""
+    for runtime in runtime_list:
+        if runtime not in INIT_RUNTIMES:
+            if not error_message:
```

Comment:
If there are multiple not-supported runtimes, it seems the debug message will still be shown repeatedly?